### DISCUSSION TITLE
Vercel.app + Facebook post redirect domain

### DISCUSSION THREAD

**Discussion by shairiezel** [vercel/next.js#44476] [App Router]
Help how to make website like this, any tutorial
[Facebook post](https://www.facebook.com/103896792542712/posts/pfbid02mfkr7osuHQ6wSk77TzdNbx8NpT52yfYVyDeirkdP5MXtpB5paJUFx2oJ3mSB1CZRl/?mibextid=Nif5oz) 

![Screenshot_20230101_153021](https://user-images.githubusercontent.com/121410786/210163826-3a59399e-5411-4efa-95f1-24fd99a03750.jpg)

**Comment by edowutv** [comment]
how do i do it?

**Reply by iAmLeViO** [reply]
I can assist you set it up on your blog, but it must be a wordpress blog but its a paid task
you can reach out to me on +234 9063 245407

**Reply by briann-rotich** [reply]
Here's how for WordPress,
**Viral Facebook post Trick using Vercel Mask URL**

1. You need just WordPress website and github account. 
2. Setup your github account to host **vercel** **FB** code(I'll provide in case you need)
3.  Point the vercel FB code in Github to vercel domain and copy the domain link. 
4. Post the vercel link with your webpage slag with square images to Facebook, easy and your post will get very high reach sometimes millions on Facebook.
If you want to learn this tutorial step by step setup you can check out my course with Viral posts proofs

Link : [Watch YouTube video Tutorial ](https://youtu.be/TRt30a_nnMI)

**Comment by 62f** [comment]
https://taqlidx877.vercel.app/m919191919?fbclid=IwAR0N0etg9rCCW_pG4oLoxE9bZQ_HaO3dVR5LTvzJS0t2BE7A0-zGWW6r2Vg
Like so:
https://l.facebook.com/l.php?u=https%3A%2F%2Ftaqlidx877.vercel.app%2Fm919191919&h=AT3WHUn-nBlWQpVdSSXt0aNERibgcS02PLoALkI-wji_g0C0w3a00KRdsraXDWE-GvAypkVn96H9-rmeVGPlBgIrvLuoj11SRpIb65iQ2o9V8bYkVD52TRdALezov75WHrhRiMHWuygvw8NyQPtC0pzrRb58Zzo0SbLr0MzMgTL-7zSra0ISl2GDVeZc_u6fdpWfr1tUkYNTx5vmCffQP-hy1TCkTAY5W6-jZ7hn9FeM-UVcS6yc7BkIecF2dcrhjv5slJEm6GFcpvIa6RqWbh87AY5UoU0xxiic0GjBakL66mdfHPQ-Jd6FaX5UMWi1UFz8NTKv3Ocksx0ERPFnVDCyED54G2Sy-byUC-YU4CmDL2iQ3A0JI-9DBDcEiqKaC10R_z4kUeQyHovkgAVy_V1hk_LpjT4f5ZGDCVjqo22ZSL5k6beB15ev9BJeSYGQBRrF4RJ-8bVWA9htb8qHS3tdQ05XqxLx5fvvB6e3OADVPFQa7yi9IMOuI65qphXnH1n9RCRLoy41_vTcBhx46N4NcWkDvOiR2Cm9JEB4GhLjJWfILHbHfL-4V6L40E8OWQ-6S7E
On the dark web?

**Reply by briann-rotich** [reply]
Here's how,
**Viral Facebook post Trick using Vercel Mask URL**

1. You need just WordPress website and github account. 
2. Setup your github account to host **vercel** **FB** code(I'll provide in case you need)
3.  Point the vercel FB code in Github to vercel domain and copy the domain link. 
4. Post the vercel link with your webpage slag with square images to Facebook, easy and your post will get very high reach sometimes millions on Facebook.
If you want to learn this tutorial step by step setup you can check out my course with Viral posts proofs

Link : [Watch YouTube video Tutorial ](https://youtu.be/TRt30a_nnMI)

**Comment by maliot123** [comment]
its easy to redirect link from vercel to facebook but how does the guy make any post go viral thats veird ive been trying 3 domains till now there must be an in code for this magic to be happened

**Reply by briann-rotich** [reply]
Here's how,
**Viral Facebook post Trick using Vercel Mask URL**

1. You need just WordPress website and github account. 
2. Setup your github account to host **vercel** **FB** code(I'll provide in case you need)
3.  Point the vercel FB code in Github to vercel domain and copy the domain link. 
4. Post the vercel link with your webpage slag with square images to Facebook, easy and your post will get very high reach sometimes millions on Facebook.
If you want to learn this tutorial step by step setup you can check out my course with Viral posts proofs

Link : [Watch YouTube video Tutorial ](https://youtu.be/TRt30a_nnMI)

**Comment by simonrangi** [comment]
How do we setup for blogger?

**Reply by briann-rotich** [reply]
Not Possible, Here Tutorial for WordPress

**Viral Facebook post Trick using Vercel Mask URL**

1. You need just WordPress website and github account. 
2. Setup your github account to host **vercel** **FB** code(I'll provide in case you need)
3.  Point the vercel FB code in Github to vercel domain and copy the domain link. 
4. Post the vercel link with your webpage slag with square images to Facebook, easy and your post will get very high reach sometimes millions on Facebook.
If you want to learn this tutorial step by step setup you can check out my course with Viral posts proofs

Link : [Watch YouTube video Tutorial ](https://youtu.be/TRt30a_nnMI)

**Comment by briann-rotich** [comment]
**Viral Facebook post Trick using Vercel Mask URL**

1. You need just WordPress website and github account. 
2. Setup your github account to host **vercel** **FB** code(I'll provide in case you need)
3.  Point the vercel FB code in Github to vercel domain and copy the domain link. 
4. Post the vercel link with your webpage slag with square images to Facebook, easy and your post will get very high reach sometimes millions on Facebook.
If you want to learn this tutorial step by step setup you can check out my course with Viral posts proofs

Link : [Watch YouTube video Tutorial ](https://youtu.be/TRt30a_nnMI)

**Comment by zeroteam10** [comment]
hello sir, can you give me a tutorial on the correct method, because I have tried it and it hasn't worked, please guide me

**Reply by briann-rotich** [reply]
Here's the Full Tutorial to get started, I've shown every step in the video Tutorials . I have also added 5 tips to make every post go Viral. Link: https://brianrotich.gumroad.com/l/zzdsb

**Reply by 62f** [reply]
Anybody got $20 I can borrow...
